classic-commerce.php

```diff
@@ -21,29 +21,48 @@
 }
 
 /**
- * Returns error when WooCommerce is detected among the files on the server.
+ * Shows an error message when WooCommerce is detected as currently active.
+ *
+ * WooCommerce and Classic Commerce cannot both be active at once.
  *
  * @return void
  */
 function cc_wc_already_active_notice() {
-	echo '<div class="error notice is_dismissible"><p>';
+	echo '<div class="notice error is-dismissible">';
+	echo '<p style="font-weight: bold">';
```

Comment:
Why not wrap this in <strong>. It is consistent with the other code. Also avoids the different ways browsers handle this and personal settings of font-weight:bold text.